agnpy/absorption/tests/test_absorption.py

```diff
@@ -228,6 +228,31 @@ def test_abs_dt_vs_point_source(self):
         # requires a 10% deviation from the two SED points
         assert check_deviation(nu, tau_dt, tau_ps_dt, 0.1)
 
+    def test_abs_blr_cubepy(self):
```

Comment:
the test as it is written now does not test much: only if you do not have a crash because of not matching API, or some internal inconsistency in the code. 
Instead you could run a test of comparing tau_blr_cubepy output with tau_blr